### PR TITLE
feat(chat): add default value in prompt variables (Issue #1592)

### PR DESCRIPTION
**Pull request by IlyaBondar**
**Description:**

add default value in prompt variables

Issues:

- Issue #1592 (part 1)

**Checklist:**

- [x] the pull request name complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] the pull request name starts with `fix(<scope>):`, `feat(<scope>):`, `feature(<scope>):`, `chore(<scope>):`, `hotfix(<scope>):` or `e2e(<scope>):`. If contains breaking changes then the pull request name must start with `fix(<scope>)!:`, `feat(<scope>)!:`, `feature(<scope>)!:`, `chore(<scope>)!:`, `hotfix(<scope>)!:` or `e2e(<scope>)!:` where `<scope>` is name of affected project: `chat`, `chat-e2e`, `overlay`, `shared`, `sandbox-overlay`, etc.
- [x] the pull request name ends with `(Issue #<TICKET_ID>)` (comma-separated list of issues)
- [x] I confirm that do not share any confidential information like API keys or any other secrets and private URLs

### CODE REVIEW

**Comment by IlyaBondar** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1726.nightly-test.deltixhub.io
>E2E tests status: success

**Comment by IlyaBondar** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-1726.nightly-test.deltixhub.io
>E2E tests status: success

File: apps/chat/src/utils/app/prompts.ts [hunk 1]
```diff
@@ -19,20 +19,26 @@ export const regeneratePromptId = (prompt: PartialBy<Prompt, 'id'>): Prompt => {
   return prompt as Prompt;
 };
 
+export const PROMPT_VARIABLE_REGEX = /{{([^|]+?)(\|.*?)?}}/g;
```
Comment (Alexander-Kezik) [inline apps/chat/src/utils/app/prompts.ts]: Maybe lets move this constant to constants folder?

**Comment by IlyaBondar** [comment]
/deploy-review
>- **Deployment failed**: [canceled](https://gitlab.deltixhub.com/Clients/deltix/nightly-test/dial/review/-/pipelines/493986)

**Comment by IlyaBondar** [comment]
/deploy-review
>- **Deployment failed**: [failed](https://gitlab.deltixhub.com/Clients/deltix/nightly-test/dial/review/-/pipelines/493990)
>Environment URL: https://chat-ai-dial-chat-pr-1726.nightly-test.deltixhub.io
>E2E tests status: success

**Review by Alexander-Kezik — APPROVED** [review]

**Pull request merged**